packages/sync-actions/test/cart-discounts-sync.spec.js

```diff
@@ -0,0 +1,289 @@
+import cartDiscountsSyncFn, {
+  actionGroups,
+} from '../src/cart-discounts'
+import { baseActionsList } from '../src/cart-discounts-actions'
+
+describe('Exports', () => {
+  it('action group list', () => {
+    expect(actionGroups).toEqual(['base'])
+  })
+
+  it('correctly defined base actions list', () => {
+    expect(baseActionsList).toEqual([
+      { action: 'changeIsActive', key: 'isActive' },
+      { action: 'changeName', key: 'name' },
+      { action: 'changeCartPredicate', key: 'cartPredicate' },
+      { action: 'changeSortOrder', key: 'sortOrder' },
+      { action: 'changeValue', key: 'value' },
+      { action: 'changeRequiresDiscountCode', key: 'requiresDiscountCode' },
+      { action: 'changeTarget', key: 'target' },
+      { action: 'setDescription', key: 'description' },
+      { action: 'setValidFrom', key: 'validFrom' },
+      { action: 'setValidUntil', key: 'validUntil' },
+    ])
+  })
+})
+
+describe('Actions', () => {
+  describe('changeIsActive action', () => {
+    let actual
+    let expected
+    beforeEach(() => {
```

Comment:
I noticed the `beforeEach` in all the tests, and the declaring of `actual` and `expected` variables above.

While it might look _good and beneficial_. It's a bit anti-pattern to the way the tests are written in this repo.

It's good if the pattern in each project is considered when contributing, I think that's the *good thing* about patterns after all.